PROCESS.md

```diff
@@ -4,7 +4,7 @@ This document describes the branching and work process adopted by the TaSecureAp
 
 ## Branching
 
-The TaSecureApi gerrit repository managed in RDK Central will employ a feature branching development
+The TaSecureApi GitHub repository managed in RDK Central will employ a feature branching development

```

Comment:
This describes the old RDK-M work flow and branching process when this repo was part of their CFM repo system. I recommend updating this document to describe our current work flow and branching strategy.

Reply:
Done